src/layers/domain/core/ods_code.py

```diff
@@ -0,0 +1,14 @@
+import re
```

Comment:
i thought we agreed that we don't need to validate ods codes any more, as the domain can assume that ods codes are valid?